packages/nimble-components/src/list-option/index.ts

```diff
@@ -51,6 +52,17 @@ export class ListOption extends FoundationListboxOption {
             .map(node => node.textContent?.trim())
             .join(' ');
     }
+
+    public override connectedCallback(): void {
```

Comment:
I have some mild concern that we'll break other components that contain list options like the combobox. Could you take a look at their test coverage and ensure it's good enough to be confident we're not breaking anything? The obvious codepath of concern is when the parent is not a `ListOptionOwner`. Presumably `connectedCallback()` just runs and does nothing. But do we want to ensure that works for dynamic cases like the tests you added to `select.spec.ts`?

Reply:
So, I've updated the `Combobox` to actually implement `ListOptionOwner`. The only other case of interest is the `RichTextMentionListbox`. If the `connectedCallback` implementation were to have a problem with non-`ListOptionOwner` types, then at the moment, I would expect Chromatic tests to fail (which they don't). Do you feel this is sufficient?

Reply:
I also did some manual testing of the rich text mention listbox and didn't see any issues. LGTM!

Reply:
Created an issue to track the tech debt we had discussed of the rich text list box not aligning with our other listbox implementations: https://github.com/ni/nimble/issues/1926